internal/monitors/device-updater/device_updater.go

```diff
@@ -24,12 +27,12 @@ type DeviceUpdater struct {
 	devStore   service.DeviceStore
 }
 
-func NewDeviceUpdater(log logrus.FieldLogger, db *gorm.DB, store *store.Store) *DeviceUpdater {
+func NewDeviceUpdater(log logrus.FieldLogger, db *gorm.DB, stores *store.Store) *DeviceUpdater {
 	return &DeviceUpdater{
 		log:        log,
 		db:         db,
-		fleetStore: store.GetFleetStore(),
-		devStore:   store.GetDeviceStore(),
+		fleetStore: stores.GetFleetStore(),
+		devStore:   stores.GetDeviceStore(),
```

Comment:
if the store abstracts the DB then should this remain singular?

```suggestion
		fleetStore: store.GetFleet(),
		devStore:   store.GetDevice(),
```